spandecorator/handler.go

```diff
@@ -72,3 +83,39 @@ func shouldIgnore(key string) bool {
 
 	return false
 }
+
+func decorateWithBody(r *http.Request, span middleware.Span) (err error) {
+	if r.Body == nil || r.Body == http.NoBody {
+		return nil
+	}
+
+	var logBody io.ReadCloser
+
+	logBody, r.Body, err = drainBody(r.Body)
+	if err != nil {
+		return fmt.Errorf("unable to extract body from request: %w", err)
+	}
+
+	//Limit tag value to 5000 characters(limit in datadog)
+	b, err := io.ReadAll(io.LimitReader(logBody, 5000)) //nolint: gomnd
+	if err != nil {
+		return fmt.Errorf("unable to read body from request: %w", err)
+	}
+
+	span.AddStringAttribute("http.request.body", string(b))
```

Comment:
Are we using constants from the internal package for the "http.request.body" for others?